packages/bitcoin-earn-fee-provider/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@liquality/bitcoin-earn-fee-provider",
+  "umdName": "BitcoinEarnFeeProvider",
+  "umdExport": "default",
+  "version": "0.4.28",
+  "description": "",
+  "module": "lib/index.js",
+  "main": "dist/index.cjs.js",
+  "files": [
+    "dist",
+    "lib"
+  ],
+  "scripts": {
+    "build:node": "webpack --config ../../webpack/webpack.node.config.js",
+    "build:browser": "webpack --config ../../webpack/webpack.browser.config.js",
+    "build": "webpack --config ../../webpack/webpack.config.js"
+  },
+  "author": "Liquality <info@liquality.io>",
+  "license": "MIT",
+  "dependencies": {
+    "@babel/runtime": "^7.4.3",
+    "@liquality/provider": "^0.4.28",
+    "axios": "^0.19.2"
+  },
+  "engines": {
+    "node": "~8.12.0"
+  },
+  "homepage": "https://github.com/liquality/chainabstractionlayer#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/liquality/chainabstractionlayer.git"
+  },
+  "bugs": {
+    "url": "https://github.com/liquality/chainabstractionlayer/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "gitHead": "1855d9ec369c8c577f74eb04e310a7c00b9eaa6c"
```

Comment:
`gitHead` shouldn't be here. Did you abort version bump?

Reply:
I usually do not version bump until its in the branch (Dev). Does that cause an issue?